pkg/api/v1/helpers.go

```diff
@@ -454,3 +455,70 @@ type NodeResources struct {
 	// Capacity represents the available resources of a node
 	Capacity ResourceList `protobuf:"bytes,1,rep,name=capacity,casttype=ResourceList,castkey=ResourceName"`
 }
+
+func (rc *ReplicationController) DecodeNestedObjects(d runtime.Decoder) error {
```

Comment:
Hrm, this wasn't what this method was intended to do.  It's not technically wrong, but I'll need to think about it.

In any case, the methods don't belong in this file, they'll need to be in something specific like decode.go.